src/Cdn/Cdn/Az.Cdn.psd1

```diff
@@ -81,13 +81,14 @@ CmdletsToExport = 'Get-AzCdnProfile', 'Get-AzCdnProfileSsoUrl', 'New-AzCdnProfil
                'New-AzCdnEndpoint', 'Unpublish-AzCdnEndpointContent', 
                'Remove-AzCdnEndpoint', 'Set-AzCdnEndpoint', 'Start-AzCdnEndpoint', 
                'Stop-AzCdnEndpoint', 'Test-AzCdnCustomDomain', 
-               'Get-AzCdnCustomDomain', 'New-AzCdnCustomDomain', 
-               'Remove-AzCdnCustomDomain', 'Enable-AzCdnCustomDomain', 
```

Comment:
@hytao why is `Enable-AzCdnCustomDomain` being removed? Removing this cmdlet would be a breaking change, which is not allowed for any of the upcoming releases.

Reply:
This cmdlet is marked as deprecated so I think it would be better to remove it.

I could add it back if removing is not allowed